docs/backend-requests/handling/manual-jwt.mdx

```diff
@@ -7,37 +7,20 @@ description: Learn how to manually verify Clerk-generated session tokens (JWTs).
+To verify the request you should check that the `isSignedIn` attribute of the `clerkClient.authenticateRequest(request)` return value is `true`. The `request` parameter provided can be any object extending `Request`.
 
-1. Using the Backend API in JSON Web Key Set (JWKS) format at the following endpoint [https://api.clerk.com/v1/jwks](https://clerk.com/docs/reference/backend-api/tag/JWKS#operation/GetJWKS).
-
-2. Using the Frontend API in JSON Web Key Set (JWKS) format at the following endpoint `https://<YOUR_FRONTEND_API>/.well-known/jwks.json`. This can be obtained from the Clerk Dashboard on the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page. Scroll down and click on **Advanced** and in the **JWT public key** section, copy the **JWKS URL**.
+The `clerkClient.authenticateRequest(request)` executes the following steps:
 
-<Images
-  src="/docs/images/backend-requests/jwt-public-key.jpg"
-  width={3024}
-  height={1650}
-  alt="The 'API Keys' page in the Clerk Dashboard. There is a red box around the 'JWT public key' section, with a red arrow pointing to the 'JWKS URL' tab of the code example."
-/>
-
-3. Using the PEM public key provided in the Clerk Dashboard on the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page. Scroll down and click on **Advanced** and in the **JWT public key** section, copy the **PEM public key**. This option should only be used as a fallback for when the first two options are not available.
-
-### Verify the token signature
-
-To verify the token signature, you should:
-
-1. Use the above Public Key to verify the token's signature.
-2. Validate that the token is not expired by checking the `exp` ([Expiration time](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.4)) and `nbf` ([Not before](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.5)) claims.
+1. Uses the instance Public Key to verify the token's signature (to manually find and use the public key check [here](#get-your-instances-public-key)).
+2. Validates that the token is not expired by checking the `exp` ([Expiration time](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.4)) and `nbf` ([Not before](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.5)) claims.
 3. The `azp` claim in the Clerk Session JWT stands for authorized parties. If the `azp` claim exists, validate that it equals any of your known origins that are permitted to generate those tokens. This is an extra security check that we highly recommend that you do. Verifying the `azp` claim on the server side ensures that the session token is generated from the expected frontend application. For example, if you are permitting tokens retrieved from `http://localhost:3000`, then the `azp` claim should equal `http://localhost:3000`. You can also pass an array of strings like so: `['http://localhost:4003', 'https://clerk.dev']`. If the `azp` claim does not exist, then you can skip this step.
```

Comment:
does authenticateRequest() do verify the `azp` claim for the user as well?

Reply:
No it doesn't. We could remove this. 
Since verifying that the jwt is not expired and the jwt is signed by the Clerk instance with the secretKey set in the developer server, is sufficient.